src/igvfd/schemas/award.json

```diff
@@ -99,26 +57,24 @@
         },
         "pi": {
             "title": "P.I.",
-            "description": "Principle Investigator of the grant.",
+            "description": "Principal Investigator(s) of the grant.",
             "comment": "See user.json for available identifiers.",
-            "type": "string",
-            "linkTo": "User"
-        },
-        "rfa": {
-            "title": "RFA",
-            "description": "The name of the bioproject phase.",
-            "comment": "The order of enums below reflects the priority when we consider selecting default analysis for a dataset.",
-            "type": "string",
-            "enum": [
-                "IGVF",
-                "community"
-            ]
+            "type": "array",
+            "uniqueItems": true,
+            "items": {
+                "title": "Investigator",
+                "description": "User object of the investigator.",
+                "type": "string",
+                "linkTo": "User"
+            }
         },
         "project": {
             "title": "Project",
             "description": "The collection of biological data related to a single initiative, originating from a consortium.",
```

Comment:
While we are at this simplification.  What does this description mean?

Reply:
Would it make more sense to say "User account of the investigator"?

Reply:
Odd, I do remember responding to this comment, but it is not here. 

I think that I understand now and this does not need to be changed